core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportCopyNumberSegmentData.java

```diff
@@ -82,7 +82,8 @@ public void testImportSegmentDataNewStudy() throws Exception {
 
         String[] args = {
         		"--data","target/test-classes/segment/data_cna_hg19.seg",
-        		"--meta","target/test-classes/segment/meta_cna_hg19_seg.txt"        		
+        		"--meta","target/test-classes/segment/meta_cna_hg19_seg.txt",
+        		"--loadMode", "bulkLoad"
```

Comment:
@morungos : never mind. We discussed it here and decided to keep your solution in this case. So, just leave the `loadMode` there